File: TUDiReader/Group.m
//
//  Group.m
//  TUDiReader
//
//  Created by Martin Weissbach on 11/3/14.
//  Copyright (c) 2014 Martin Weissbach. All rights reserved.
//

#import "Group.h"
#import "PersistenceStack.h"

@implementation Group

@dynamic feeds;
@dynamic name;

- (NSArray *)orderedFeeds
{
    return [self.feeds sortedArrayUsingDescriptors:@[[NSSortDescriptor sortDescriptorWithKey:@"title" ascending:YES]]];
}

+ (NSEntityDescription *)entityDescription
{
    return [NSEntityDescription entityForName:@"Group" inManagedObjectContext:[PersistenceStack sharedPersistenceStack].managedObjectContext];
}

@end

File: TUDiReaderToday/Group.swift
//
//  Group.swift
//  TUDiReader
//
//  Created by Martin Weissbach on 1/20/15.
//  Copyright (c) 2015 Martin Weissbach. All rights reserved.
//

import Foundation
import CoreData

class Group : NSManagedObject {
    
    @NSManaged var name: String
    @NSManaged var feeds : NSSet
    
}